Changes

```diff
@@ -4,6 +4,9 @@ Working version
 
 ### Runtime system:
 
+- #10236 Add a runtime warning when allocating a float array.
```

Comment:
I would add that the warning can be controlled by `Sys.enable_float_array_alloc_warning` or `OCAMLRUNPARAM=F=n`, as it may not be evident where to look for this warning otherwise.

Reply:
I think the parameter is currently bugged, let me fix this.